website/blog/2023-11-13-OAI-assistants/index.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: AutoGen Meets GPTs
+authors: gagb
+tags: [openai-assistant]
+---
+
+![OpenAI Assistant](img/teaser.jpg)
+<p align="center"><em>AutoGen enables collaboration among multiple ChatGPTs for complex tasks.</em></p>
+
+
+## TLDR
+OpenAI assistants are now integrated into AutoGen via [`GPTAssistantAgent`](https://github.com/microsoft/autogen/blob/main/autogen/agentchat/contrib/gpt_assistant_agent.py).
+This enables multiple OpenAI assistants, which form the backend of the now popular GPTs, to collaborate and tackle complex tasks.
+Checkout example notebooks for reference:
+* [Basic example](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_oai_assistant_twoagents_basic.ipynb)
+* [Code interpreter](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_oai_code_interpreter.ipynb)
+* [Function calls](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_oai_assistant_function_call.ipynb)
+
+
+## Introduction
+Earlier last week, OpenAI introduced [GPTs](https://openai.com/blog/introducing-gpts), giving users ability to create custom ChatGPTs tailored for them.
+*But what if these individual GPTs could collaborate to do even more?*
+Fortunately, because of AutoGen, this is now a reality!
+AutoGen has pioneered agents and multi-agent workflows [since before](https://aka.ms/autogen-pdf) the launch of GPTs.
+And now we are introducing compatibility with the [Assistant API](https://openai.com/blog/introducing-gpts), which is currently in beta preview.
```

Comment:
Mention AutoGen's version number starting from which this feature is supported, i.e., 0.2.0b5?